__tests__/app.test.js

```diff
@@ -217,3 +217,55 @@ describe("GET /api/articles", () => {
       });
   });
 });
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of comments for the given article ID with mentioned properties", () => {
+    const article_id = 3;
+    return request(app)
+      .get(`/api/articles/${article_id}/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeInstanceOf(Array);
```

Comment:
Must include a check for the length of the array, rather than an instance of an array -  otherwise you could get a false positive from your forEach